extensions/wikia/PageShare/PageShareController.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class PageShareController extends WikiaController {
+
+	/**
+	 * @var PageShareHelper
+	 */
+	private $helper;
+
+	public function __construct() {
+		parent::__construct();
+		$this->helper = new PageShareHelper();
```

Comment:
you use helper only in prepareShareServices method. Are you sure about creating it in constructor instead inside the method you actually use it in?